frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/package.json

```diff
@@ -18,14 +18,21 @@
     "react-router-dom": "5.3.0"
   },
   "dependencies": {
+    "@babel/runtime": "^7.24.7",
```

Comment:
necessary ? if so, can you move this to dev dependencies
also provide a little info on why this lib was added